org/cloudfoundry.yml

```diff
@@ -2402,6 +2402,7 @@ orgs:
             - KevinJCross
             - silvestre
             - marcinkubica
+            - olivermautschke
```

Comment:
New approvers should not be added anymore to the deprecated, non-standard github teams. These teams will be deleted very soon by #262.

Adding to the WG charter is sufficient. Org automation will add you to https://github.com/orgs/cloudfoundry/teams/wg-app-runtime-interfaces-autoscaler-approvers. 

Not adding to team 'App Runtime Interfaces WG Autoscaler' also helps also to find remaining hidden usage of this team before deletion (like Concourse authorisation).

Reply:
@OliverMautschke Can you please remove the changes to `org/cloudfoundry.yml` from this PR?